sydent/sydent.py

```diff
@@ -305,7 +306,7 @@ def parse_config(config_file):
         config_file (str): the file to be parsed
     """
 
-    cfg = ConfigParser.SafeConfigParser()
+    cfg = configparser.ConfigParser()
```

Comment:
I think this will do the wrong thing on py2? ConfigParser doesn't have a `.set` method there

Reply:
As per discussion in #synapse-dev, sticking with `ConfigParser` is fine.